src/App.css

```diff
@@ -18,19 +43,225 @@ h1 {
   margin-right: 5rem;
 }
 
+.navbar li a {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 400;
+  text-decoration: none;
+  color: #121212;
+  font-size: 0.8rem;
+  letter-spacing: 1.9px;
+}
+
+.circle {
+  border: 1px solid #e8e8e8;
+  height: 2.8rem;
+  width: 2.8rem;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ] It will be nice to also adhere to the DRY principle with this highlighted property.